modules/social_features/social_topic/social_topic.module

```diff
@@ -267,7 +268,8 @@ function social_topic_widget_alter(array &$form) {
 
   // Change the widget if more than X.
   if (count($form['field_topic_type']['widget']['#options']) >= $config->get('social_topic_type_select_changer')) {
-    $form['field_topic_type']['widget']['#type'] = 'select';
+    $form['field_topic_type']['widget']['#type'] = 'select2';
+    $form['field_topic_type']['widget']['#multiple'] = TRUE;
```

Comment:
@dmytriikaiun  I think it would be better to retrieve cardinality from the field `field_topic_type` instead of only `TRUE`. It will prevent conflict issues if someone will revert the cardinality of `field_topic_type` to `1` on their own site. What do you think about it?